posthog/api/element.py

```diff
@@ -105,16 +91,10 @@ def stats(self, request: request.Request, **kwargs) -> response.Response:
                 "type": elements[2],
                 "elements": [ElementSerializer(element).data for element in chain_to_elements(elements[0])],
             }
-            for elements in result[:limit]
+            for elements in result
         ]
 
-        if paginate_response:
```

Comment:
how does sampling remove the need for pagination?

Reply:
I only added pagination to try and let folk that were willing to wait longer to get more data do just that. But the feedback was roughly "it's nice to see the data, but annoying to click the button"

So, I'm hoping that sampling means we can draw the heatmap without having to page. 🤔 May be safer to leave pagination in and have a larger page. Then I can experiment a little more with the impact.